tests/drivers/build_all/led/app.overlay

```diff
@@ -104,6 +104,26 @@
 				compatible = "onnn,ncp5623";
 				reg = <0xd>;
 			};
+
+			is31fl3194@e {
+				compatible = "issi,is31fl3194";
+				reg = <0xe>;
+				led_rgb {
+					label = "led_rgb";
+					color-mapping = <0>, <1>, <2>;
```

Comment:
Eventually, you can use the `LED_COLOR_ID_x` notations.

Reply:
The overall approach in the `build_all` tests is to compile without semantic consideration,
so I want to leave intentionally make these parameters "nonsense" here as well.

Reply:
OK, but in `tests/drivers/build_all/led_strip/app.overlay` you used the `LED_COLOR_ID_x` notation... I am fine with both approaches but they must be consistent :)

Reply:
Thank you for your comment.

Yes, I am aware of that...

The code already in led_strip was written like that, so I followed that.
But I think this PR way is the correct way to handle build_all...
There seems to be some difference of opinion among developers on this point...

Reply:
Since I have no opinion it is fine by me. Thanks for the explanation.